docs/tutorial/accelerator_design.md

```diff
@@ -0,0 +1,25 @@
+# Accelerator Design
+
+## TODOs:
+- Describe the accelerator of interest
+- Show a detailed block diagram of the signals
+- Mention and describe what the CSR Manager does
+- Describe the CSRs to be used
+    - Mode: +, -, x, XOR
+    - Data length
+    - Start
+    - Busy
+    - Counter
+- Have a timing diagram interface to describe what happens for the signals
+- Indicate which repository to see this structure
+
+## Accelerator Architecture
+
+### Modes and Features
+
+### Interfaces
+
+### CSR Manager
```

Comment:
This section has it's own file?

Reply:
I updated this! but locally. We will separate the CSR manager part not this one.

Reply:
You will see it in the next PR 😄